### PR TITLE
Mini ui fixes

### PR DESCRIPTION
**Pull request by schloo**

### CODE REVIEW

File: app/templates/carpools/show.html [hunk 1]
```diff
@@ -2,78 +2,163 @@
...
-    </li>
-{% endfor %}
-</ul>
-</div>
 
-{% else %}
-<a class="btn btn-default" href="{{ url_for('carpool.new_rider', carpool_uuid=pool.uuid) }}" role="button">Request a seat</a>
-{% endif %}
+            <div class="two-col-layout top-border">
+                <h4 class=>Driver's Corner: Passenger Requests</h4>
```
Comment (iandees) [inline app/templates/carpools/show.html]: Missing class here.

File: app/templates/carpools/show.html [hunk 2]
```diff
@@ -2,78 +2,163 @@
...
+
+                <ul>
+
+                {% for request in pool.get_ride_requests_query() %}
+
+                    <li><strong>{{ request.person.name }}</strong>
+                        <br>Gender: {{ request.person.gender }}
+                        <br>Notes: {{ request.notes }
```
Comment (iandees) [inline app/templates/carpools/show.html]: Missing last `}` here.

File: app/templates/carpools/show.html [hunk 3]
```diff
@@ -2,78 +2,163 @@
...
+                        </small>
+                    </li>
+
+                {% endfor %}
+
+                </ul>
+            </div>
+
+            <div class="two-col-layout">
+                <div class="two-col-column"></div>
```
Comment (iandees) [inline app/templates/carpools/show.html]: Empty divs?

File: app/templates/carpools/show.html (hunk 1 above)
Comment (schloo) [reply app/templates/carpools/show.html]: whoops! moving too fast without being able to run locally this morning! fixed.

File: app/templates/carpools/show.html (hunk 2 above)
Comment (schloo) [reply app/templates/carpools/show.html]: fixed

File: app/templates/carpools/show.html (hunk 3 above)
Comment (schloo) [reply app/templates/carpools/show.html]: fixed

File: app/templates/carpools/show.html [hunk 4]
```diff
@@ -2,78 +2,153 @@
...
+
+            <h4>Carpool Status</h4>
+
+        {% if current_user_ride_request and current_user_ride_request.status == 'approved'  %}
+            <div class="two-col-layout">
+                <div class="two-col-column">
+                    <p>You are confirmed for this carpool!</p>
+                </div>
+                <div class="two-col-column">
+                    <a href="{{ url_for('carpool.modify_ride_request', carpool_uuid=pool.uuid, request_uuid=current_user_ride_request.uuid, action='cancel') }}"><input class="destructive" type="submit" value="Cancel your seat in carpool"></a>
```
Comment (iandees) [inline app/templates/carpools/show.html]: These `<input>`s inside of `<a href>`s won't work. They need to be styled `<a href>`.

Comment (iandees) [reply app/templates/carpools/show.html]: (They won't work because the browser will just go back to the carpool details page without doing anything.)

**Comment by iandees** [comment]
@schloo I changed the `<input>`s to `<a>`s with Bootstrap `btn` styles. If that doesn't work or make sense, please open up a new PR.

**Pull request merged**